src/i18n/en.json

```diff
@@ -1524,5 +1524,10 @@
     "membersPage.memberNav.manageMember": "Manage member",
     "membersPage.memberNav.makePayment": "Make payment",
     "membersPage.memberNav.copyWalletAddress": "Copy wallet address",
-    "widget.totalActions": "Total actions"
+    "widget.totalActions": "Total actions",
+    "balancePage.burger.label.addFunds": "Add funds",
```

Comment:
I know it's an older PR, if you have the time, could you please take these strings into the component where they're being rendered? as per our [standards](https://github.com/JoinColony/ColonyFrontEndLivingStandard/blob/master/codeQuality/index.md#i18n)